src/constants/default-settings.js

```diff
@@ -694,3 +694,6 @@ export const MAP_CONFIG_DESCRIPTION = 'Map config will be included in the Json f
 export const BASE_SPEED = 600;
 export const DEFAULT_TIME_FORMAT = 'MM/DD/YY HH:mm:ssa';
 export const SPEED_CONTROL_RANGE = [0, 10];
+
+// No Op
+export const NO_OP = () => {};
```

Comment:
do we really need to add a N0_OP to default settings? feel like it's a little overkill, adding one line of import every file to use no_op probably not that pretty

Reply:
this was removed